example/driver/simpledriver.go

```diff
@@ -179,6 +179,11 @@ func (s *SimpleDriver) HandleReadCommands(deviceName string, protocols map[strin
 		} else if reqs[0].DeviceResourceName == "Zrotation" {
 			cv, _ := sdkModels.NewCommandValue(reqs[0].DeviceResourceName, common.ValueTypeInt32, s.zRotation)
 			res[0] = cv
+		} else if reqs[0].DeviceResourceName == ".rotation" || reqs[0].DeviceResourceName == "[A-Z]+rotation" {
```

Comment:
Are we requiring each device service to implement RegEx support???
I thought the SDK would be querying the device resources using the RegEx and then pass the matched list of resource to `HandleReadCommands` 

I don't think this current approach meets the need that was originally discussed.

Reply:
discussed with @cloudxxx8 and I'll make the change and implement the RegEx support in SDK instead of device service layer.